lib/net/nvnetdrv/nvnetdrv.c

```diff
@@ -79,37 +79,34 @@ static KINTERRUPT g_interrupt;
 static HANDLE g_irqThread;
 static KEVENT g_irqEvent;
 
-// Manage RX ring
-static volatile struct descriptor_t *g_rxRing;
-static KSEMAPHORE g_rxRingFreeDescriptors;
-static HANDLE g_rxRingRequeueThread;
+// Manage RX and TX rings
+static struct descriptor_t *g_rxRing;
+static struct descriptor_t *g_txRing;
+static size_t g_rxRingSize;
+static size_t g_txRingSize;
 static size_t g_rxRingHead;
-static size_t g_rxRingTail;
+static size_t g_txRingHead;
+static atomic_size_t g_txPendingCount;
+static atomic_size_t g_rxPendingCount;

```

Comment:
The type of `g_rxPendingCount` seems to have been changed accidentally here

Reply:
Fixed